variables.tf

```diff
@@ -463,3 +463,39 @@ variable "graphdb_enable_userdata_scripts_on_reboot" {
   type        = bool
   default     = false
 }
+
+#cmk building in progress
+variable "enable_cmk" {
+  description = "Enable CMK for encryption. If false, use AWS managed key."
+  type        = bool
+  default     = false
+}
+
+variable "kms_master_key_id" {
+  description = "ARN of the Customer Managed Key (CMK)."
+  type        = string
+  default     = ""
+}
+
+variable "policy" {
```

Comment:
Can we name it cmk_policy for example?

Reply:
changed to sns_cmk_policy. It is currently not being used anyway, since it is mostly for if one wants to add their own policy. I guess I could remove it also?